lib/src/layout_element.dart

```diff
@@ -112,6 +112,9 @@ class TableLayoutElement extends LayoutElement {
           columni++;
         }
         if (child is TableCellElement) {
+          String width = child.attributes["width"] ?? "";
```

Comment:
You'll have to remove this change if you want to have your PR merged, as it is not really what you are trying to fix.

Reply:
Thanks for your checking, fixed now.